pkg/provider/vsphere/plugin.go

```diff
@@ -104,15 +117,15 @@ func (p *plugin) Provision(spec instance.Spec) (*instance.ID, error) {
 		}
 	}
 
-	newInstance, err := parseParameters(properties, p)
+	newVM, err := parseParameters(properties, p)
 	if err != nil {
-		log.Errorf("Error: \n%v", err)
+		log.Error("Problems Whilst Parsting", "Error", err)
```

Comment:
Sorry a nit pick here.  We haven't had a convention for logging, so maybe now is the time to start...  All the logging statements elsewhere use `"err"` as a key for error.  Would you mind changing to that?

Reply:
Sure, I did find the logging a little confusing (lack of fatal etc.) a convention would help
 :-)